### PR TITLE
[term entry]: d3-scaling range

### LINKED ISSUE #3887: [Term Entry] D3 Scaling .range()
### Reason/inspiration (optional)

We would like a new term entry on the `.range()` method for [D3](https://www.codecademy.com/resources/docs/d3): Scaling. The entry should go in a new file under `docs/content/d3/concepts/scaling/terms/range/range.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```js `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by vrun1208**
### Description

closes #3887 
Added new term entry - d3 Scaling .range()

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by dakshdeepHERE — APPROVED** [review]
Looks good for second review. 🚀

File: content/d3/concepts/scaling/terms/range/range.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+```pseudo
+d3.scaleType()
+  .domain([minValue, maxValue])
+  .range([minOutput, maxOutput]);
+```
+
+- **.scaleType():** The selected scale method (e.g., `.scaleLinear()`- a continuous linear scale).
+- ••.domain():** Requires the `minValue` and `maxValue` arguments to set the accepted range.
+- **.range():** The second step in defining a scale, the arguments `minOutput` and `maxOutput` are used to specify the set of values returned by the scale.
```
Comment (avdhoottt) [inline content/d3/concepts/scaling/terms/range/range.md:28]:
```suggestion
- `.scaleType()`: The selected scale method (e.g., `.scaleLinear()`- a continuous linear scale).
- `.domain()`: Requires the `minValue` and `maxValue` arguments to set the accepted range.
- `.range()`: The second step in defining a scale, the arguments `minOutput` and `maxOutput` are used to specify the set of values returned by the scale.
```
Use backticks to indicate methods or reserved keywords. Change the last line as it contains plagiarism (Line of `.range()`)

File: content/d3/concepts/scaling/terms/range/range.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+```js
+const scale = d3.scaleLinear().domain([0, 100]).range([0, 500]);
+
+console.log(scale(0));
+console.log(scale(50));
+console.log(scale(100));
+```
+
+The above example will give the following output:
```
Comment (avdhoottt) [inline content/d3/concepts/scaling/terms/range/range.md:42]:
```suggestion
The above example will give the following output:
```
Change the line a little bit as it contains some plagiarism.

**Review by avdhoottt — CHANGES_REQUESTED** [review]
@vrun1208 Great entry, I have suggested some minor changes, please make them.

File: content/d3/concepts/scaling/terms/range/range.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+Tags:
+  - 'Methods'
+  - 'Data'
+  - 'Graphs'
+CatalogContent:
+  - 'learn-d3'
+  - 'paths/computer-science'
+---
+
+The **`.range()`** method is used to define the output range of a scale. The range represents the output values produced by the scale when mapped from the `domain`. The syntax for the `.range()` method depends on the type of scale used.
```
[the remark this answers is not in the document]
Comment (vrun1208) [reply content/d3/concepts/scaling/terms/range/range.md:16]: Yes! I wrote that one. how to link this?

[the remark this answers is not in the document]
Comment (dakshdeepHERE) [reply content/d3/concepts/scaling/terms/range/range.md:16]: use the markdown style to link it like [this](https://www.codecademy.com/resources/docs/d3/scaling/domain)
image

[the remark this answers is not in the document]
Comment (vrun1208) [reply content/d3/concepts/scaling/terms/range/range.md:16]: okay got it 👍

**Review by avdhoottt — APPROVED** [review]
Ready to merge!!

**Pull request merged**